tests/Doctrine/Tests/Annotations/AbstractReaderTest.php

```diff
@@ -408,6 +408,26 @@ public function testInvalidAnnotationButIgnored()
         self::assertEmpty($reader->getPropertyAnnotations($class->getProperty('foo')));
     }
 
+    public function testGloballyIgnoredAnnotationNotIgnored()
```

Comment:
` : void` missing

Reply:
I'm not sure how's that going in line with all ither code in this file. Isn't it better to put these things aside? Like there's #177 which is supposed to do exactly what you're asking, but all over all code.

Reply:
@sanmai I understand your frustration here, these are **cosmetic** things and don't have much impact on the behaviour of the code. However we have an agreement among maintainers to try as much as possible to make sure our CS is applied everywhere, even when the project isn't following it properly (that's why we have automated incremental checks on ORM and DBAL, for example).

We haven't had the chance of applying the incremental checks here and applying things on the entire code base is a bit too much (as explained in #177), hence my comments.

I'm sorry for the confusion and will completely understand if you decide to not apply the suggestions in the comments, I'm just trying to ensure that **we** (Doctrine team) follow **our** agreements. A maintainer is able to edit your commit on your fork (if you hadn't changed the option while creating the PR).

I hope you'll understand the motivation and I thank you for your contribution.

Reply:
Well, that's just me that spend a good amount of time to make sure that my code looks just like every other code, but if there's a consensus to write newer code in a more appropriate style, I'm good with that too. I'd change on my part but I'd like if these new guidelines would be somewhere at least. Then anyone else wouldn't need to spend the time so useless as I did. Are we on the same page with this?

Reply:
@lcobucci anything else?

Reply:
> I'd change on my part but I'd like if these new guidelines would be somewhere, at least.

That one is documented here: https://www.doctrine-project.org/projects/doctrine-coding-standard/en/latest/reference/index.html#project-level-ruleset ("Always add native types where possible")

Reply:
Well, that's not exactly obvious, what do you think? How am I supposed to find it?..

A like to these guidelines from the README.md, or even from CONTRIBUTING.md, would be best.

Reply:
I think you're right! I might make PRs to change that if you do not beat me to it.

Reply:
@greg0ire see #239